Sources/ActionTrait.php

```diff
@@ -27,7 +27,7 @@ trait ActionTrait
 	 * An instance of this class.
 	 * This is used by the load() method to prevent multiple instantiations.
 	 */
-	protected static self $obj;
+	protected static $obj;
```

Comment:
Why do we need to do this?

Reply:
PHPStorm says `Property type does not match` because `static self` and `@var static` are not the same.

Reply:
Interesting, I would have said correct the @var as the documentation above says it should be an instance of the class.